README.rst

```diff
@@ -671,7 +746,31 @@ Preprocessed images
 Group-level
 ===========
 
--  ``grouplevel/...``
+-  |  For task-based group-level analyses, HALFpipe outputs the statistical maps for the effect, the variance, the degrees of freedom of the variance, and the t-statistic and z-statistic for each linear model that is specified, much as for the first-level features. These models are fit per session, task, first-level feature, first-level task contrast, and group-level contrast, where these exist.
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_stat-effect_statmap.nii.gz``
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_stat-variance_statmap.nii.gz``
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_stat-dof_statmap.nii.gz``
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_stat-t_statmap.nii.gz``
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_stat-z_statmap.nii.gz``
+   |
+-  |  For every contrast, the mean and standard deviation of the regressor used in the analysis.
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_mean.nii.gz``
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_std.nii.gz``
+   |
+-  |  For every contrast, the corresponding brain mask is output beside the statistical maps. Masks do not differ between different features calculated, they are only copied out repeatedly for convenience
```

Comment:
I would be very clear here that the masks can differ if different sub-groups/inclusion criteria are used

Reply:
Resolved